x-pack/plugins/apm/common/projections/rum_overview.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  Setup,
+  SetupTimeRange,
+  SetupUIFilters,
+  // eslint-disable-next-line @kbn/eslint/no-restricted-paths
+} from '../../server/lib/helpers/setup_request';
+import { PROCESSOR_EVENT } from '../elasticsearch_fieldnames';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { rangeFilter } from '../../server/lib/helpers/range_filter';
```

Comment:
I know you didn't introduce this but it just occurs to me that we should never import server code into common (the imports above are fine since it's only types which will disappear at runtime).

Would you mind moving `rangeFilter.ts` into commons, if it's not too difficult? If your editor supports refactorings it should be able to update all the import statements for you.

Reply:
addressed in [https://github.com/elastic/kibana/pull/68086/commits/f1c9bcf7cd15674d375824323f98db355aaf0240 ](https://github.com/elastic/kibana/pull/68086/commits/f1c9bcf7cd15674d375824323f98db355aaf0240)